README.md

```diff
@@ -6,6 +6,22 @@ PlaceCal is a large and very complicated app for collating organisation and even
 
 To get an idea of the project and what we're about, check out [the handbook](https://handbook.placecal.org/) which is also [in it's own respository](https://github.com/geeksforsocialchange/PlaceCal-Handbook), and especially the [Glossary](glossary.md).
 
+## Requirements
+
+To run Placecal locally you will need:
+
+- a mac or a linux machine
+- gcc
+- postgres 
+  - if you are running in a docker image you will still need the dev libraries for the gem to compile against
```

Comment:
I think you need the postgres developer libraries regardless of whether you run in Docker or not? (It will use the same interface if you are running over TCP or via UNIX Domain Sockets) -- I would suggest also just making this a little bit more explicit that you mean:
- The postgres developer libraries (specifically libpq-dev (debian) / postgresql-libs (arch) / whatever)
- Postgres is the thing that is being dockerized in this example, not placecal :)